pycheribuild/projects/run_qemu.py

```diff
@@ -563,13 +564,37 @@ class LaunchFreeRTOSQEMU(LaunchQEMUBase):
     _enable_smbfs_support = False
     _add_virtio_rng = False
 
+    default_demo = "RISC-V-Generic"
+    default_demo_app = "main_blinky"
+
     @classmethod
     def setup_config_options(cls, **kwargs):
         super().setup_config_options(defaultSshPort=None, **kwargs)
 
+        cls.demo = cls.add_config_option(
+            "demo", metavar="DEMO", show_help=True,
+            default=cls.default_demo,
+            help="The FreeRTOS Demo to run (default: RISC-V-Generic)")  # type: str
```

Comment:
default should be appended automatically.

Reply:
It's appended automatically, as in if you don't provide that flag it will run RISC-V-Generic, if that's what you mean?

Reply:
I meant the default value will automatically be appended to the --help message.